packages/blocks/src/api/validation/index.js

```diff
@@ -644,11 +652,11 @@ export function getBlockContentValidationResult( blockTypeOrName, attributes, or
  * @param {string|Object} blockTypeOrName      Block type.
  * @param {Object}        attributes           Parsed block attributes.
  * @param {string}        originalBlockContent Original block content.
- *
+ * @param {string}        mode                 Validation mode.
  * @return {boolean} Whether block is valid.
  */
-export function isValidBlockContent( blockTypeOrName, attributes, originalBlockContent ) {
-	const { isValid } = getBlockContentValidationResult( blockTypeOrName, attributes, originalBlockContent, createLogger() );
+export function isValidBlockContent( blockTypeOrName, attributes, originalBlockContent, mode = 'exact-match' ) {
```

Comment:
a very minor thing, but I wonder if 'strict' might be better terminology instead of 'exact-match', as with the likes of https://github.com/WordPress/gutenberg/issues/18666, the current validation approach is trying to move away from an exact match as such.

Reply:
Thanks updated in https://github.com/WordPress/gutenberg/pull/19188/commits/4e3c867e95a362f96b81c119264d37e7451983b7 to "strict" and "no-save-error" respectively. Happy to work on names further after I get some good examples in.